code/modules/mob/living/carbon/xenomorph/castes/spiderling/spiderling.dm

```diff
@@ -0,0 +1,94 @@
+/mob/living/carbon/xenomorph/spiderling
+	caste_base_type = /mob/living/carbon/xenomorph/spiderling
+	name = "Spiderling"
+	desc = "Smoll spoder" // will change later
+	icon = 'icons/Xeno/Effects.dmi'
+	icon_state = "spiderling walking"
+	health = 200
+	maxHealth = 200
+	plasma_stored = 200
+	pixel_x = 0
+	old_x = 0
+	tier = XENO_TIER_MINION
+	upgrade = XENO_UPGRADE_BASETYPE
+	pull_speed = -2
+	flags_pass = PASSXENO
+
+/mob/living/carbon/xenomorph/spiderling/Initialize(mapload, mob/living/carbon/xenomorph/spidermother)
```

Comment:
this is an issue btw, having to call parent here.

You don't want spiderling to have verbs, they shouldn't have nicknumber, they shouldn't be counted towards xeno created tally, the list goes on and on

Reply:
I pass on that for the TM, but I think there are bunch of stuff to fix here